requirements.txt

```diff
@@ -1,5 +1,6 @@
 Babel==2.6.0
 chardet==3.0.4
+cn2an==0.5.8
```

Comment:
@d-fence Do we need anything else here in master?

Reply:
I'm sorry to say that I cannot accept this new requirement two reasons:

1. l10n_cn is an optional module and this requirements will be installed even if you don't install l10n_cn
2. AFAIK there is no Debian package for cn2an

Instead, you should try the import of cn2an and handle the case when the import fails
Look [here](https://github.com/odoo/odoo/blob/6abd50ae8e1b19339163c717705826544e222e13/addons/http_routing/geoipresolver.py#L5-L8) for example: